packages/jsii-pacmak/lib/targets/go/runtime/property-access.ts

```diff
@@ -14,12 +15,24 @@ export class GetProperty {
 
   public emit(code: CodeMaker) {
     const resultVar = slugify('returns', [this.parent.instanceArg]);
+    const implMapVar = slugify('implMap', [this.parent.instanceArg]);
     code.line(`var ${resultVar} ${this.parent.returnType}`);
+    code.line(`${implMapVar} := make(${JSII_IMPL_MAP_TYPE})`);
+
+    const implMap =
+      this.parent.reference?.scopedImplMap(this.parent.parent.pkg) ?? [];
+    if (implMap.length) {
+      const [interfaceName, structName] = implMap;
+      code.line(
+        `${implMapVar}[reflect.TypeOf((*${interfaceName})(nil)).Elem()] = reflect.TypeOf((*${structName})(nil)).Elem()`,
+      );
+    }
```

Comment:
Same as before - is there any reason we cannot use a literal here?

And since this map is fed with respects to `this.parent.reference?`, can we not have this be computed *once* and stored somewhere for re-use?